pkg/scclient/client.go

```diff
@@ -981,3 +993,32 @@ func (c *RegistryClient) startBackOff(microServiceID string, callback func(*Micr
 		return
 	}
 }
+
+func (c *RegistryClient) checkSCIHealth() {
+	timeOut := time.Duration(1) * time.Second
+	for _, v := range c.Config.Addresses {
+		conn, err := net.DialTimeout("tcp", v, timeOut)
+		if err != nil {
+			c.Config.Status[v] = unavailable
+		} else {
+			c.Config.Status[v] = available
+			conn.Close()
+		}
+	}
+}
+
+func (c *RegistryClient) sciMonitor() {
+	ticker := time.NewTicker(5 * time.Second)
```

Comment:
5s is so frequency

Reply:
how long is appropriate, 25s ? 30s ?

Reply:
imagine you got 1000 service instances, each one dial each sc server instance's port every 5 seconds that is 1000*sc instances connections．
default should be 30s
and can be configurable

Reply:
agree, but have some trouble
1. if can be configurable, should pass the option with r.Initialize(sco)
2. I found there are one service instance may create three RegistryClient instances "r.Initialize(sco)"

Reply:
ok， use env “CHASSIS_SC_HEADLTH_CHECH_INTERVAL=30s”